app/controllers/application_controller.rb

```diff
@@ -32,16 +23,28 @@ def redirect_to_apigateway
     redirect_to "#{ENV['APIGATEWAY_URL']}/auth?callbackUrl=#{auth_login_url}&token=true"
   end
 
+  def logout_apigateway
+    redirect_to "#{ENV['APIGATEWAY_URL']}/auth/logout?callbackUrl=#{auth_login_url}&token=true"
+  end
+
+  def access_denied(exception)
```

Comment:
Doesn't this just render a Json instead of redirecting to the login webpage?

Reply:
It does, because since the login webpage is not in the application, there's no way for me to flash a message saying that there was something wrong with the login. Do you know if it's possible for me to flash an error message and then redirect to the login page?